man/gemfile.5.ronn

```diff
@@ -287,7 +300,14 @@ Git repositories support a number of additional options.
   * `branch`, `tag`, and `ref`:
     You `MUST` only specify at most one of these options. The default
     is `:branch => "master"`
+  * For example:
```

Comment:
Don't use a code block, these need to be indented by 8 spaces and space separate them between each line